### ISSUE TITLE
Context Path not being read from application.properties file

### ISSUE THREAD

**Issue by AkhtarShahid** [stormpath/stormpath-spring-boot-war-example#2]
-I added an application.properties file in resource folder which has contextPath defined in that like 
server.contextPath=/db/makeCall. But when i deploy the application in local tomcat then it ignores context Path and just takes war file name in context path. But the same application when i use with embedded tomcat then it takes the contexPath from application.properties file.

**Comment by nagkumar** [comment]
yes, all the tomcat properties (including port, contextpat etc..) set in application.properties are applicable only to embedded tomcat, which is used when bootRun target is used..
